ci-operator/config/konveyor/crane-runner/konveyor-crane-runner-main.yaml

```diff
@@ -0,0 +1,46 @@
+base_images:
+  cli:
+    name: "4.11"
+    namespace: ocp
+    tag: cli
+  ubi:
+    name: ubi
+    namespace: ocp
+    tag: "8"
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.16
+images:
+- dockerfile_path: Dockerfile
+  from: ubi
+  inputs:
```

Comment:
Skipping for now adding `crane` container image as an input for the sake of getting into openshift/release. That'll be taken care of in a follow-up PR.